tools/topology/m4/detect_coef.m4

```diff
@@ -0,0 +1,18 @@
+# initial coe params for detector.
```

Comment:
Where is the ABI for this defined ? please reference it here.

Reply:
and this comment makes no sense

Reply:
@lgirdwood This is not finalized yet as it depends on the real detect algorithm, just a placeholder at the moment.

Reply:
So it's a testing value/pattern for this PR ? Should it be upstreamed then ?

Reply:
@lgirdwood you are right, let me send it to integration branch only.
@ranj063 extra lines fixed, thanks.